README.md

```
@@ -159,25 +97,20 @@ This library uses the following Hugging Face datasets:
 
 The data processing pipeline downloads these datasets and stores them in the `data_dir` specified during the `DataDecide` initialization.  Then does some filtering, parsing, merging, and pulling in external information about hpms and other training settings.
 
+The ingest CLI defaults to `~/data` and can be overridden by setting `DATADEC_DATA_DIR`.
+
 ## Project Structure
 
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)